### DISCUSSION TITLE
found an apple script to enable unsigned extension

### DISCUSSION THREAD

**Discussion by scratchusernamemrtbts** [ScratchAddons/ScratchAddons#1328] [Show and tell]
i have an update for #340 
find a way to make apple script enable unsigned extension
just have to enter your password
[tmp.scpt.zip](https://github.com/ScratchAddons/ScratchAddons/files/5819038/tmp.scpt.zip)
finding a way to make it automatically run when open safari

**Comment by WorldLanguages** [comment]
Saying again just in case, AFAIK it won't work properly as a Safari extension even if you sort this out because browser.cookies onchange event is not supported in Safari.

**Reply by scratchusernamemrtbts** [reply]
oh!
let's wait until apple safari support it
